draco/analysis/flagging.py

```diff
@@ -233,23 +234,36 @@ def process(self, ss):
         ss.redistribute("freq")
 
         baselines = self.telescope.baselines
+
+        # The masking array. True will *retain* a sample
```

Comment:
I guess we should flag this to switch conventions in the future for consistency.

Reply:
Good point. It's only internal so I can just changing the naming to make it clearer.

Reply:
Okay. I've done that, but it was slightly invasive, so it would be good if you could look again.